### ISSUE TITLE
still error when using YAML.load() with callback

### ISSUE THREAD

**Issue by m80126colin** [jeremyfa/yaml.js#18]
As title, here's my error message.

```
c:\Users\user\Documents\GitHub\others\train\node_modules\yamljs\bin\yaml.js:1589

                value = value.split("\r\n").join("\n").split("\r").join("\n");
                              ^
TypeError: Object  has no method 'split'
    at Object.YamlParser.cleanup (c:\Users\user\Documents\GitHub\others\train\no
de_modules\yamljs\bin\yaml.js:1589:17)
    at Object.YamlParser.parse (c:\Users\user\Documents\GitHub\others\train\node
_modules\yamljs\bin\yaml.js:980:21)
    at Object.Yaml.parse (c:\Users\user\Documents\GitHub\others\train\node_modul
es\yamljs\bin\yaml.js:240:15)
    at c:\Users\user\Documents\GitHub\others\train\node_modules\yamljs\bin\yaml.
js:215:24
    at c:\Users\user\Documents\GitHub\others\train\node_modules\yamljs\bin\yaml.
js:309:22
    at fs.js:266:14
    at Object.oncomplete (fs.js:107:15)
```

**Comment by jeremyfa** [comment]
Interesting. I think you are trying to read a file that doesn't exist. That said, the error should not be the one you are experiencing.

I will take a look when I have a bit of time.

**Comment by jeremyfa** [comment]
The newly released version `v0.2.0` should fix it.
